src/qml/components/SearchContactsList.qml

```diff
@@ -0,0 +1,180 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import "../base"
+import "../components"
+import "../theme"
+
+Item {
+    id: searchContactsList
+    property real headerHeight: 20
+    property real headerOpacity: 0
+    property alias contactListView: contactListView
+    property alias modelCount: contactListView.count
+
+    anchors {
+        top: contactSearch.bottom
+        left: parent.left
+        right: parent.right
+        bottom: parent.bottom
+        topMargin: 1
+        bottomMargin: defaultChatHeight
+        margins: defaultMargins
+    }
+
+    state: {
+        if (search !== controllers.users.userId) {
+            if (models.discoveredContacts.newContactFiltered) {
+                return "show header"
+            } else {
+                return "hide header"
+            }
+        }
+        return "hide header"
+    }
+
+    states: [
+        State {
+            name: "show header"
+            PropertyChanges {
+                target: searchContactsList
+                headerHeight: 90
+                headerOpacity: 1
+            }
+        },
+        State {
+            name: "hide header"
+            PropertyChanges {
+                target: searchContactsList
+                headerHeight: 20
+                headerOpacity: 0
+            }
+        }
+    ]
+
+    transitions: [
+        Transition {
+            NumberAnimation {
+                properties: "headerHeight, headerOpacity"
+                easing.type: Easing.InExpo
+                duration: Theme.animationDuration
+            }
+        }
+    ]
+
+    ListView {
+        id: contactListView
+        signal placeholderClicked()
+
+        anchors.fill: parent
+        model: models.discoveredContacts.proxy
+        spacing: 10
```

Comment:
spacing/margin/padding/colors are defined in Theme.qml, don't hurry to hardcode new integer